### DISCUSSION TITLE
Beta version of QRControl 1.8.2 is out under Google Play Store.

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#259] [Announcements]
While the user facing changes in 1.8.2 are minor, there was enough internal overhaul to release as a beta for a short while.

Additions: Support for new 2023 Extensions: GCMP, WBLK, WIDE 
Added command set control (making some subtle command style changes.)
Removed: UX Pro Mode/Easy mode from the first page, as Labs handles this mostly automatically.  Can be manual set from the Extra tab. 
Bug fix: better support for larger fonts
Bug fixes: null safety overhaul

Android: https://play.google.com/store/apps/details?id=com.miscdata.qrcontrol

There will be an iOS release soon, just no as a beta, Apple made that too difficult last time a tried.

**Comment by andrerav** [comment]
Awesome! I'm so stoked for GCMP and WIDE. Going to test the heck out of those!

**Reply by dnewman-gpsw** [reply]
HERO11 2.10.70 should be out in a few days.

**Comment by DemonViewLLC** [comment]
Hey, just joined up, didn't know where else to report this.  Looks like QR control got updated on my Android on the 11th.  However when I open it, the screen on my phone immediately goes dark.  I tried to adjust it with the brightness level on the Android phone, but it says that it cant adjust it as the brightness level is being controlled by the app.  Using a Samsung S20FE.  Have restarted the phone, but that didn't work.  Will follow up with clearing the cache, data and then do a clean install.  Just thought I'd let you know though.

**Reply by DemonViewLLC** [reply]
Ok, clearing the data worked to solve this issue.  Screen is once again bright enough to see.
